### ISSUE TITLE
Update to spack-stack v1.9.2

### ISSUE THREAD

**Issue by GeorgeGayno-NOAA** [NOAA-EMC/UFS_UTILS#983]
It will be installed on all RDHPCS machines. The plan is to have the 'classic' and LLVM-compiled versions side-by-side. The global workflow will start with the 'classic' version.

The location of the stack on each machine is [here](https://gist.github.com/DavidHuber-NOAA/a75daea63a81959dba456424c644740e).

Note: the move to spack-stack v1.9.2 on Ursa was made during the port to that machine. See #1064. So, this issue will update Orion and Hercules.

Related to #879.

**Comment by DavidHuber-NOAA** [comment]
@GeorgeGayno-NOAA The new target for this issue is spack-stack v1.9.1.  Would you mind updating the title to reflect that?  Thanks 😃

**Comment by DavidHuber-NOAA** [comment]
@GeorgeGayno-NOAA the GW team is working on porting to 1.9.1 and Ursa across the submodules.  Would you be able to add support for 1.9.1?  I will note that `ip` has fully acquired all of the functionality of `sp`, so the latter should be removed.

Also, spack-stack 1.9.1 was not installed on Jet, S4, or Gaea C5 as NESDIS and EPIC have dropped support for these systems.  The global workflow has removed support for Jet and S4 and is not maintaining/will soon likely drop support for C5.

**Comment by GeorgeGayno-NOAA** [comment]
> [@GeorgeGayno-NOAA](https://github.com/GeorgeGayno-NOAA) the GW team is working on porting to 1.9.1 and Ursa across the submodules. Would you be able to add support for 1.9.1? I will note that `ip` has fully acquired all of the functionality of `sp`, so the latter should be removed.
> 
> Also, spack-stack 1.9.1 was not installed on Jet, S4, or Gaea C5 as NESDIS and EPIC have dropped support for these systems. The global workflow has removed support for Jet and S4 and is not maintaining/will soon likely drop support for C5.

Yes, I can add support for 1.9.1. Are there any instructions for loading the modules on each machine?

**Comment by DavidHuber-NOAA** [comment]
Great, thank you!  Here are the locations of the official installations:

Hera: `/contrib/spack-stack/spack-stack-1.9.1/envs/ue-oneapi-2024.2.1`
Hercules and Orion: `/apps/contrib/spack-stack/spack-stack-1.9.1/envs/ue-oneapi-2024.1.0`
Gaea C6: `/ncrc/proj/epic/spack-stack/c6/spack-stack-1.9.1/envs/ue-2023.2.0`
Ursa: `/contrib/spack-stack/spack-stack-1.9.1/envs/ue-oneapi-2024.2.1`

I don't have a full listing of library versions being updated yet, but I can update the following list as I work through other submodules:

EDIT I made a [gist](https://gist.github.com/DavidHuber-NOAA/a75daea63a81959dba456424c644740e) instead.

**Comment by GeorgeGayno-NOAA** [comment]
The use of spack stack v1.9.2 on Ursa will be handled via #1068 (and support for Hera will end).

**Comment by DavidHuber-NOAA** [comment]
@GeorgeGayno-NOAA I will be opening a PR for this today, though I won't be able to test it everywhere.  Would you be able to help with that or assign someone to it?

**Comment by GeorgeGayno-NOAA** [comment]
> [@GeorgeGayno-NOAA](https://github.com/GeorgeGayno-NOAA) I will be opening a PR for this today, though I won't be able to test it everywhere. Would you be able to help with that or assign someone to it?

@BrianCurtis-NOAA and I can help with testing.
